### PR TITLE
Games details screen

### PR DESCRIPTION
**Pull request by Khumoyiddin**
[//]: # (Feel free to remove parts that do not relate to the changes you are willing to bring into main)

### Games details screen
[//]: # (Give a brief summary of why the change is being implemented, i.e. feature was built, bug fixing and etc.)

### Changes
[//]: # (List change details here for Code Reviewers)
1. Added list view controller
2. Added all text and images inside list view controller

### CODE REVIEW

**Comment by Khumoyiddin** [comment]
Same situation, can't add to reviewers Abboskhon Shukurullaev

File: steam_app/controllers/Games/GamesViewController.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 import UIKit
```
Comment (abubakrj) [inline steam_app/controllers/Games/GamesViewController.swift]: Extra line

File: steam_app/controllers/Games/GamesViewController.swift [hunk 2]
```diff
@@ -27,7 +28,9 @@ class GamesViewController: UIViewController {
 
 extension GamesViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        print("tapping on cell")
+        tableView.deselectRow(at: indexPath, animated: true)
+        performSegue(withIdentifier: "showDetail", sender: self)
+//        navigationController?.pushViewController(GamesDetailsViewController(), animated: false)
```
Comment (abubakrj) [inline steam_app/controllers/Games/GamesViewController.swift]: You can remove unused line of code

**Review by abubakrj — COMMENTED** [review]
@Khumoyiddin Can you add a screenshot of the end result?

**Comment by Khumoyiddin** [comment]
Screenshot 2023-01-10 at 10 10 14
The screenshot of game details screen

File: steam_app/controllers/Games/GamesViewController.swift (hunk 1 above)
Comment (Khumoyiddin) [reply steam_app/controllers/Games/GamesViewController.swift]: done

File: steam_app/controllers/Games/GamesViewController.swift (hunk 2 above)
Comment (Khumoyiddin) [reply steam_app/controllers/Games/GamesViewController.swift]: done

**Comment by AbboskhonShukurullaev** [comment]
It is better to add images in the Assets

Screenshot 2023-01-10 at 13 18 43

**Comment by AbboskhonShukurullaev** [comment]
Which one of them are tappable? If there is something tappable, it should be button, not a label/imageView

Screenshot 2023-01-10 at 13 22 51

**Comment by AbboskhonShukurullaev** [comment]
Colors should not be different

Screenshot 2023-01-10 at 13 24 44

**Comment by Khumoyiddin** [comment]
> Which one of them are tappable? If there is something tappable, it should be button, not a label/imageView
> 
> Screenshot 2023-01-10 at 13 22 51

There are nothing tapable. That's why there is label or imageView is used

**Comment by AbboskhonShukurullaev** [comment]
Too much space

Screenshot 2023-01-10 at 23 25 28

**Comment by AbboskhonShukurullaev** [comment]
Still not fixed when tapping. Tint color Should not be shawn. 

Screenshot 2023-01-10 at 23 26 47

**Review by AbboskhonShukurullaev — APPROVED** [review]

**Pull request merged**